core/crop/cropping.go

```diff
@@ -0,0 +1,102 @@
+package crop
+
+import (
+	"github.com/otrego/clamshell/core/point"
+)
+
+//CroppingPreset : Convenience enum for specifying a cropping direction.
```

Comment:
Nit: Typically there's a space between the comment chars // and the actually comment. Ex:

```go
// CroppingPreset
```